library/core/class.locale.php

```diff
@@ -36,6 +36,9 @@ class Gdn_Locale extends Gdn_Pluggable implements LocaleInterface {
      */
     private $addonManager = null;
 
+    /** @var \Vanilla\Contracts\ConfigurationInterface $gdnConfig */
+    private $gdnConfig;
```

Comment:
`$gdnConfig` is a pretty odd name for something that might not be a `Gdn_Config` at all.


Maybe just `config`?